dashboard/db/schema.rb

```diff
@@ -849,7 +849,7 @@
     t.string   "school_type",                   null: false
     t.datetime "created_at",                    null: false
     t.datetime "updated_at",                    null: false
-    t.string   "address_line1",      limit: 30,              comment: "Location address, street 1"
+    t.string   "address_line1",      limit: 50
```

Comment:
It looks like the `change_column` migration causes the column to lose its comment - perhaps it needs to be copied into the migration to preserve it?

Reply:
Updated.